### ISSUE TITLE
CLI issues

### ISSUE THREAD

**Issue by mathieumg** [Cellule/npm-lifecycle-runner#1]
I'm trying to use the new CLI approach from `0.1.0` but I'm hurting a wall, whenever I try to run the following command:

``` js
node_modules/.bin/npm-lifecycle-runner
```

I get the following error: `: No such file or directory`

I thought it could have been because of the `#!/usr/bin/env node` in the cli file but I'm using other CLI tools right beside this one that use the same pattern without any issues.

For what it's worth, I'm on Ubuntu 12.04 x64, Node v0.10.29 and npm 2.4.2 (yapm).

**Comment by Cellule** [comment]
What exactly is your use case ?
The main use case is to have the following in your package.json(5). 

``` js
  "scripts": {
    "install": "npm-lifecycle-runner \"Echo Some command I want to run in dev only\" \"Echo some other command\""
  },
```

**Comment by mathieumg** [comment]
Yeah that's what I started with (script hook), but I narrowed it down to what I reported above.

If running `node_modules/.bin/npm-lifecycle-runner` or `node_modules/.bin/npm-lifecycle-runner "echo test"` doesn't work, there is no way it would work when used inside a script hook.

**Comment by Cellule** [comment]
Well for starters, it shouldn't work to run it by itself.
See https://github.com/Cellule/npm-lifecycle-runner/blob/master/installScript.js#L3
On the other hand you should receive a readable error...
I know that the binary is a symlink on linux and not on windows. I wonder if that might be the reason

**Comment by Cellule** [comment]
I attempted a fix on v0.1.1, but I fear it might not do the trick
Could you also try `npm install npm-lifecycle-runner@next` which doesn't install script.
I think the issue might be that npm is trying to run the install script with the runner which is not yet installed.

**Comment by mathieumg** [comment]
> On the other hand you should receive a readable error...

Yeah. If I replace the content of your `cli` file with a simple `console.log` call, it doesn't get there either, so the problem isn't with the script itself. I removed the `node_modules/.bin/npm-lifecycle-runner` file and the `node_modules/npm-lifecycle-runner` directory and then installed v0.1.1 but to no avail, still getting the `: No such file or directory` error.

Same with `@next` which points to `0.2.0-alpha.1`. I don't see what could be causing this... Perhaps it does have to do with the env, but it's still a big haystack to look in. As pointed out previously, I use other modules which have the same paradigm to expose their CLI tool and they work correctly, boggles the mind...
